resource-aggregate/cqrs/operations/operator.go

```diff
@@ -18,8 +18,8 @@ type Operator struct {
 	raClient   service.ResourceAggregateClient
 }
 
-// NewOperator instancies Operator
-func NewOperator(subscriber eventbus.Subscriber, raClient service.ResourceAggregateClient) *Operator {
+// New instancies Operator
+func New(subscriber eventbus.Subscriber, raClient service.ResourceAggregateClient) *Operator {
```

Comment:
These commands / operator is nothing else then RA Client. Why it's not part of the ResourceAggregateClient ? I would move it out from cqrs completely; has nothing to do with CQRS. It's a RA client.

Reply:
It has something with CQRS - it uses crqs events

Reply:
It consumes notifications, same as any nats client. It doesn't make him CQRS pattern aware.
Do you disagree having it part of a RAClient?